doc/mgr/smb.rst

```diff
@@ -418,6 +418,14 @@ custom_smb_global_options
     things in ways that the Ceph team can not help with. This special key will
     automatically be removed from the list of options passed to Samba.
 
+.. warning::
+   Setting the `clustering` option allows an administrator to choose exactly
+   when Samba's CTDB clustering will be used. By default, the use of Samba's
+   clustering is derived from the `placement` count.  If you choose to set
+   `clustering` make sure you understand how clustering interacts with
+   placement. In particular, be aware that running multiple instances of the
+   same smb service without clustering enabled can cause unexpected behavior.
```

Comment:
Suggest SMB uppercase, or if you intend it to be a literal service name, double backticks

Reply:
done

Reply:
*double* backticks please so it's rendered as fixed-width, not italics.  We don't have a rigid style guide like we should but this is the practice I've been trying to follow.

The PR is approved for whenever you see fit to merge.

Reply:
Sure thing. This is an .rst file, I forgot!
Fixed.

Reply:
It's not like a certain other system that uses {{text}} ;)